### ISSUE TITLE
Avoid unstylable <div> elements around Slotfills.

### ISSUE THREAD

**Issue by ryanboswell** [WordPress/gutenberg#6839]
**Is your feature request related to a problem? Please describe.**
While I really like the new `PluginPostStatusInfo` slot fill component, it's almost impossible to get rows to match the look and feel of the default panel rows in the Status & Visibility metabox because all slot fills end up with an empty `<div>` wrapped around them.

As one example, adding a custom button to perform some operation, results in the following markup at minimum (assuming the only child of `PluginPostStatusInfo` is the button):

```
<div class="components-panel__row">
    <div>
        <button type="button" class="components-button button-link ">Here is a Button</button>
    </div>
</div>
```

Meanwhile the default trash button has the following markup:

```
<div class="components-panel__row">
    <button type="button" class="components-button editor-post-trash button-link button-link-delete">Move to trash</button>
</div>
```

That extra div causes a lot of problems when trying to position or style the row contents and makes it difficult to emulate the look of other "core" rows above, thanks to the lack of a parent selector in CSS.

**Describe the solution you'd like**
Ideally the Slotfill mechanism would use something like `<Fragment>`s to wrap the filled children rather than a `<div>` so there isn't any additional "phyiscal" markup added to the page. But given that the `<div>` is there to add a DOM ref, I'm not sure that would work.

A possibly more achievable alternative would be to allow a `className` prop to be passed and set on the wrapping `<div>` so it could at least be targeted for styling:

This component code:
```
<PluginPostStatusInfo className="style-that-div">
    <MyCustomButton />
</PluginPostStatusInfo>
```

Would then be rendered to:
```
<div class="components-panel__row">
    <div class="style-that-div">
        <button type="button" class="components-button button-link ">Here is a Button</button>
    </div>
</div>
```

**Comment by danielbachhuber** [comment]
Related #6089

**Comment by gziolo** [comment]
We made logical mistake when implementing `Slot`:
```diff
+PluginPostStatusInfo.Slot = () => (
+	<PanelRow>
+		<Slot />
+	</PanelRow>
+);
````

`PanelRow` should be added for every Fill instead. I will open PR with a fix shortly.

**Comment by gziolo** [comment]
Patch is ready for review: #6867.

**Comment by gziolo** [comment]
I think I fixed the original issue with #6867, can you confirm?

**Comment by ocean90** [comment]
I can confirm that`<label>` and `<FormToggle>` are now working as expected. Thanks @gziolo
